### PR TITLE
fix(TBD-9802): Compile error in spark job with HBase on HDP 3.1

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdpx/Hortonworks_HDP_3_1_4_12_1.json [hunk 1]
```diff
@@ -1169,6 +1169,11 @@
               "id": "DYNAMIC_HDP_3_1_4_12_1_hadoop_yarn_common_3_1_1_3_1_4_12_1_jar",
               "tagName": "library"
             },
+            {
+              "childNodes": [],
+              "id": "DYNAMIC_HDP_3_1_4_12_1__hbase_server_2_0_2_3_1_4_12_1_jar",
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/builtin/hdpx/Hortonworks_HDP_3_1_4_12_1.json]: isn't there two "_" ? Did you get that regenerating the builtin ?

**Comment by aparent-talend** [comment]
Why is there no changes in the template file ?

**Comment by aparent-talend** [comment]
But I saw you generated a built-in 23 days ago (in the same PR in which you modyfied the template) and there was no double _ ?

**Comment by sponomarova** [comment]
> But I saw you generated a built-in 23 days ago (in the same PR in which you modified the template) and there was no double _ ?

Unfortunately, now it is quite hard to track all the history of this change :( 
Not sure why at some point it was generated with double "_". 
Maybe wrong branch or some untracked changes in my eclipse. My bad. Sorry about that mess.

**Pull request merged**